### PR TITLE
Creating the Docker Image for the frontend UI

### PR DESCRIPTION
**Pull request by simardeep1792**
Requesting your code review for using Deno and React Docker Image

### CODE REVIEW

File: Rs-Pack/server.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { serve } from "https://deno.land/std/http/server.ts";
```
Comment (Stephen-ONeil) [inline Rs-Pack/server.ts:1]: As is, looks like the server will use the latest version of the module every time it starts. Looks like it's officially unstable and pre-1.0, so it'd be best to lock it to the current version with `... from "https://deno.land/std@0.224.0/http/server.ts";`.

Dependabot and `npm audit` will be blind to this, unfortunately, so periodically checking for new versions, determining if there's breaking changes, and upgrading will all be a manual chore for now.

I was initially going to suggest we switch the whole project from node/npm to deno/jsr for consistency, but it looks like dependabot doesn't yet have [deno support](https://github.com/dependabot/dependabot-core/issues/2417) (or JSR, though there's not even an issue requesting it yet). It also appear's deno's own audit functionality is [stalled in discussion](https://github.com/denoland/deno/discussions/8699), and on the JSR side, again, there aren't even any issues asking for it yet.

@sleepycat you may be more aware of the Deno/JSR landscape than me, are there dependency management/auditing solutions out there that I'm missing?

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]

File: Rs-Pack/server.ts (hunk 1 above)
Comment (simardeep1792) [reply Rs-Pack/server.ts]: Thanks for the review. Just made the changes and up for review, feel free to provide feedback!

**Review by Stephen-ONeil — APPROVED** [review]
Good catch on specifying the deno container version as well. I think this is good to merge, though I still want Mike to weigh in on the maturity of the Deno/JSR dependency scanning situation at some point, before we push too hard towards it in new projects :wink:.

**Comment by sleepycat** [comment]
@Stephen-ONeil raises a good point that if it costs us vulnerability scanning this is probably not something we want showing up in other projects, but I think there is more to learn here if we treat this as an isolated experiment.
This is pretty low stakes and easy to transparently replace with an equivalent... basically a perfect experiment, so IMO we should run with it for the moment.
🚢

**Review by sleepycat — APPROVED** [review]
This looks like a great starting place to me.

**Pull request merged**